app/scripts/controllers/overview.ctrl.js

```diff
@@ -63,21 +53,6 @@ angular.module('dashboard')
 		$log.debug("overviewCtrl: showInfo");
 	};
 
-	self.toggleMenu = function() {
-        $log.debug("overviewCtrl: toggleMenu");
-		$rootScope.menu = !$rootScope.menu;
-    };
-
-	self.meetingsClicked = function() {
```

Comment:
Can't remove this (yet), otherwise he +/- button for overview meetings wont' work.